tests/unit/models/transactions/test_amm_bid.py

```diff
@@ -0,0 +1,66 @@
+from unittest import TestCase
+
+from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.models.currencies import XRP, IssuedCurrency
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions import AMMBid, AuthAccount
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_ASSET = XRP()
+_ASSET2 = IssuedCurrency(currency="ETH", issuer="rpGtkFRXhgVaBzC5XCR7gyE2AZN5SN3SEW")
```

Comment:
nit: the linting on this is weird

Reply:
Oh, should line 10 be in multiple lines?

Reply:
I'm referring to the indenting - I would expect it to be more like the one(s) below

Reply:
I ran flake on this module and it didn't complain.